schedule/migration/x86_regression_test_online.yaml

```diff
@@ -79,17 +81,12 @@ conditional_schedule:
         - console/sshd
         - console/ssh_cleanup
         - console/mtab
+        - console/upgrade_snapshots
         - console/zypper_lifecycle
         - console/orphaned_packages_check
-        - console/kdump_and_crash
```

Comment:
Please don't remove it, this module we added for kdump.

Reply:
Added